spec/features/add_space_spec.rb

```diff
@@ -1,25 +1,23 @@
-feature 'Add space' do
-  let(:name){'MY SPACE'}
-  let(:description){'description'}
-  let(:price){'12.11'}
-  let(:date){'2016-05-01'}
+feature 'Add space form:' do
+  let(:name){'first house'}
+  let(:description){'pretty'}
+  let(:price){'10.01'}
+  let(:date){'11 may 2016'}
 
   before :each do
-    named_signup
-    visit '/spaces/new'
+    visit '/'
+    click_button('Register')
+    submit_signup_form("host","host","h@m.com","host_pass","host_pass")
   end
 
-  scenario 'to table' do
-    fill_in :space_name, with: name
-    fill_in :space_description, with: description
-    fill_in :space_price, with: price
-    fill_in :space_availability, with: date
-    click_button 'Submit'
+  scenario 'new space gets listed on the main page' do
```

Comment:
Again, as mentioned yesterday I think one should not outsource EVERYTHING to the web helper. Because in order to understand a test a person would have to go back and forth between spec file and web helper. The action that is being test in my eyes should be visible the first time step by step. Then, the second time use a web helper. A person would not need to go back to the web helper because all the steps will be in one of the previous tests.